h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -394,7 +394,7 @@ public static Vec makeCon(double x, long len, int log_rows_per_chunk, boolean re
 
   public static Vec makeCon(double x, long len, int log_rows_per_chunk, boolean redistribute, byte type) {
     int chunks0 = (int)Math.max(1,len>>log_rows_per_chunk); // redistribute = false
-    int chunks1 = (int)Math.min( 4 * H2O.NUMCPUS * H2O.CLOUD.size(), len); // redistribute = true
```

Comment:
there's a bunch of those in current codebase: should I create a ticket to review usages of `H2O.NUMCPUS` and `Runtime.getRuntime().availableProcessors()` ?

Reply:
Yes, calls to Runtime.getruntime.availableProcessors() should be discarded. Let's go for H2O.NUMCPUS calls everywhere we can. That is a good change :+1: